src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+     method="post"
+     class="form"
+     data-qa="big"
+     >
```

Comment:
```suggestion
    <form
      action="#"
      method="post"
      class="form"
      data-qa="big"
    >
```
Давай використовувати такі відступи і таке форматування